lib/checkPullRequestJob.js

```diff
@@ -0,0 +1,210 @@
+ */
+const getAllChangedFiles = async (context) => {
+  const changedFiles = [];
+  /**
+   * @type {import('probot').Octokit.PullsGetResponse}
+   */
+  const pullRequest = context.payload.pull_request;
+  const maxFilesPerPage = 100;
+
+  let totalFilesFetched = 0;
+  let currentPage = 1;
+  do {
+    const fileListParams = context.repo({
+      pull_number: pullRequest.number,
+      per_page: maxFilesPerPage,
+      page: currentPage,
+    });
+
+    const fileResponse = await context.github.pulls.listFiles(fileListParams);
+    changedFiles.push(...fileResponse.data);
+    totalFilesFetched += fileResponse.data.length;
+    currentPage++;
+  } while (totalFilesFetched < pullRequest.changed_files);
+
+  return changedFiles;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ */
+const getRegistryReminder = (newJobFiles, changedFiles) => {
+  let registryReminder = ', ';
+
+  const hasNotAddedFileToRegistry = newJobFiles.some(
+    (file) => !doesRegistryContainFile(file.filename, changedFiles)
+  );
+
+  if (hasNotAddedFileToRegistry) {
+    const fileText = newJobFiles.length > 1 ? 'files' : 'file';
+    registryReminder +=
+      'endeavour to add the new job ' + fileText + ' to the job registry and ';
+  }
+
+  return registryReminder;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ */
+const getJobNameString = (newJobFiles) => {
+  let jobNameString = '';
+  const jobNameArray = newJobFiles.map((file) =>
+    getJobNameFromFileName(file.filename)
+  );
+  if (jobNameArray.length === 1) {
+    jobNameString = ' The name of the job is ' + jobNameArray[0] + '.';
+  } else {
+    jobNameString = ' The jobs are ' + jobNameArray.join(', ') + '.';
+  }
+
+  return jobNameString;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ * @param {string} prAuthor - Author of the Pull Request
+ */
+const getCommentBody = (newJobFiles, changedFiles, prAuthor) => {
+  const registryReminder = getRegistryReminder(newJobFiles, changedFiles);
+  const jobNameString = getJobNameString(newJobFiles);
+  const newLineFeed = '<br>';
+  const serverJobsForm = 'server jobs form'.link(
+    'https://goo.gl/forms/XIj00RJ2h5L55XzU2');
+
+  // This function will never be called when there are no job files.
+  if (newJobFiles.length === 1) {
+    const serverAdminPing = 'Hi @' + SERVER_JOBS_ADMIN + ', PTAL at this PR, ' +
+      'it adds a new server job.' + jobNameString;
+
+    const prAuthorPing = 'Also @' + prAuthor + registryReminder +
+    'please make sure to fill in the ' + serverJobsForm +
+    ' for the new job to be tested on the backup server. ' +
+    'This PR can be merged only after the test is successful.';
```

Comment:
Add a link to the guide on jobs:  https://github.com/oppia/oppia/wiki/Running-jobs-in-production#submitting-a-pr-with-a-new-job.

Reply:
Done.